.gitignore

```diff
@@ -26,4 +26,8 @@ docs/_build/
 tensorflow-hub
 
 # for Macs
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+# build outputs for PyPI
+build/
```

Comment:
don't we need to also remove these from the repo in addition to adding them to gitignore?

Reply:
thanks, I don't know how / why the `build/` files were in the repo before, but I removed the folder!